packaging/docker/Dockerfile

```
@@ -74,13 +74,20 @@ RUN \
     addgroup -g ${NETDATA_GID} -S "${DOCKER_GRP}" && \
     adduser -S -H -s /usr/sbin/nologin -u ${NETDATA_GID} -h /etc/netdata -G "${DOCKER_GRP}" "${DOCKER_USR}" && \
     # Apply the permissions as described in
-    # https://github.com/netdata/netdata/wiki/netdata-security#netdata-directories
-    chown -R root:netdata /etc/netdata && \
-    chown -R netdata:netdata /var/cache/netdata /var/lib/netdata /usr/share/netdata && \
-    chown -R root:netdata /usr/lib/netdata && \
-    chown -R root:netdata /usr/libexec/netdata/ && \
+    # https://docs.netdata.cloud/docs/netdata-security/#netdata-directories, but own everything by root group due to https://github.com/netdata/netdata/pull/6543
+    chown -R root:root \
+        /etc/netdata \
+        /usr/share/netdata \
+        /usr/libexec/netdata && \
+    chown -R netdata:root \
+        /usr/lib/netdata \
+        /var/cache/netdata \
+        /var/lib/netdata \
+        /var/log/netdata && \
+    chmod 0755 /usr/libexec/netdata/plugins.d/*.plugin && \
     chmod 4750 /usr/libexec/netdata/plugins.d/cgroup-network /usr/libexec/netdata/plugins.d/apps.plugin && \
-    chmod 0750 /var/lib/netdata /var/cache/netdata && \
+    # Group write permissions due to: https://github.com/netdata/netdata/pull/6543
+    chmod 0770 -R /var/lib/netdata /var/cache/netdata && \
```

Last one from me :-D

There are also files in `/var/lib/netdata` and `/var/cache/netdata`, so I think this should be:

```suggestion
    find /var/lib/netdata /var/cache/netdata -type d -exec chmod 0770 {} \; && \
    find /var/lib/netdata /var/cache/netdata -type f -exec chmod 0660 {} \; && \
```